buildSrc/src/main/kotlin/com/toasttab/protokt/shared/ProtoktBuild.kt

```diff
@@ -34,3 +37,22 @@ private fun createExtensionConfiguration(project: Project) {
         extendsFrom(extensionsConfiguration)
     }
 }
+
+private fun addProtoktCoreDependency(project: Project, ext: ProtoktExtension) {
+    if (project.name !in setOf("protokt-wkt", "protokt-core")) {
```

Comment:
"shared" build code should not have any internal protokt build logic; this state should be pulled into protokt-specific part of buildSrc and passed into configureProtokt as a variable

Reply:
Done.